src/pokeContainer.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import PokeList from './pokeList.js';
+import './Styles/pokeContainer.css';
+
+
+
+class pokeContainer extends Component {
+    state = { data: [], loading: true, query: null, sortOrder: 'asc', sortType: 'pokemon', page: 1, lastPage: 1, };
+    
+componentDidMount() {
+    this.fetchData();
+  };
+  
+  fetchData = async () => {
```

Comment:
In this fetch function, you'll want to reset the paging so if someone searches while on the last page, it resets them back to the first page to see their results